src/components/member-list-item/index.js

```diff
@@ -19,8 +57,24 @@ const PreviewMember = ({ memberDetails }) => {
       as={`/${username}`}
       key={username}
     >
-      <div className={container}>
+      <div
+        ref={cardRef}
+        className={container}
+        onMouseEnter={dev && mouseEnter}
+        onMouseLeave={dev && mouseLeave}
+      >
         <Card developerInfo={memberDetails} />
+        {dev && (
+          <div ref={divref} style={descriptionDivStyle}>
+            <p style={pstyle}>Company: {memberDetails?.company || 'nil'}</p>
+            <p style={pstyle}>
+              Designation: {memberDetails?.deesignation || 'nil'}
+            </p>
+            <p style={pstyle}>
+              Years Of Experience: {memberDetails?.yoe || 'Not Experienced'}
```

Comment:
```suggestion
              Years Of Experience: {memberDetails?.yoe || 'Just Starting Out'}
```

What do you think?
cc - @ankushdharkar